src/acc/acc_bench_smm.c

```diff
@@ -20,9 +20,12 @@
 #if !defined(ELEM_TYPE)
 # define ELEM_TYPE double
 #endif
-#if !defined(EPSILON)
+#if !defined(EPSILON) && 0
```

Comment:
This looks a bit like a leftover from debugging?
Maybe remove completely?

Reply:
Indeed, it it makes little sense to have a fixed tolerance given all parameters/problem sizes; let alone DP vs. SP. I will remove this.